packages/tools/src/tools/annotation/CircleROITool.ts

```diff
@@ -996,7 +1002,11 @@ class CircleROITool extends AnnotationTool {
           worldPos2
         );
         const isEmptyArea = worldWidth === 0 && worldHeight === 0;
-        const area = Math.abs(Math.PI * (worldWidth / 2) * (worldHeight / 2));
+        const scale = getScale(image);
+        const area =
+          Math.abs(Math.PI * (worldWidth / 2) * (worldHeight / 2)) /
+          scale /
```

Comment:
instead can we have `worldWidth/scale/2` and `worldHeight/scale/2` for readability?